### ISSUE TITLE
USER ISSUE: game crashing when trying to start a new map

### ISSUE THREAD

**Issue by frozenhammerz** [StrangeLoopGames/EcoIssues#4409]
**Version:** 0.7.1.0 beta

**Steps to Reproduce:**
1. i know i can play it, i can connect to other servers
2. when i tried to start new map for single player experience, it just crashed

**Expected behavior:**
started a new map

**Actual behavior:**
The following error message popped up:
```
Connection Failed
```

**Comment by MarjieAVolk** [comment]
Hey there - Can you attach the crash dump from your server?  They should be located in the server installation folder; in '[...]/steamapps/common/Eco Server'

**Comment by rainfarmer** [comment]
User says:
I tried logging into my own private game under "Continue" and it boots me out when the loading screen says "game playerstats...", the screen minimizes the game and then i get a Windows error message saying "Eco.exe not responding". 

Thing's I've tried:
Restarting my computer, restarting Eco, restarting the modem, verifying integrity of game files, and deleting and re-downloading Eco. I'm at a loss.
